pom.xml

```diff
@@ -294,6 +296,9 @@
         <rat.skip>true</rat.skip>
         <spotless.check.skip>true</spotless.check.skip>
         <shade.phase.prop>none</shade.phase.prop>
+        <archiver.compress>false</archiver.compress>
+        <archiver.recompressZippedFiles>false</archiver.recompressZippedFiles>
+        <npm.command>install --no-audit</npm.command>
```

Comment:
Yes, npm.command is not used anymore. 
In theory it should be possible to limit or turn off potential dependency updates of `npm install` but I don't think there's any guarantee. I used `npm install --no-audit` earlier because it takes about 1s vs ~10s taken by `npm ci`.